src/mainwindow.h

```diff
@@ -358,6 +359,8 @@ private slots:
   // when painting the count
   QIcon m_originalNotificationIcon;
 
+  size_t m_NumberOfProblems;
```

Comment:
Maybe make this atomic (`std::atomic<std::size_t>`)?

Reply:
Addressed